app/controllers/songs_controller.rb

```diff
@@ -18,19 +17,17 @@ def create
   end
 
   def show
+    @comment = Comment.new
+  end
+
+  rescue_from ActiveRecord::RecordNotFound do
```

Comment:
nghĩ rộng bài toán ra, các case em gặp phải để tìm ra xem mình nên đặt ở đâu. 
1. Xem lại về OOP kế thừa 
2. Tìm xem lỗi này có thể xảy ra ở đâu nữa 

từ đó sẽ có giải pháp đặt đâu là tốt nhất